de-webapp/src/main/java/org/iplantc/de/Application.java

```diff
@@ -93,7 +93,7 @@ public void onStartup(ServletContext container) {
         dispatcher.setLoadOnStartup(0);
         dispatcher.addMapping("/de/websocket");
         dispatcher.setInitParameter("org.atmosphere.cpr.packages", "org.iplantc.de.server");
-        LOG.error("************** Started AtmosphereServlet ***********");
+        LOG.info("************** Started AtmosphereServlet ***********");
```

Comment:
Not necessary change for this issue but it kept bothering me so changed the log level.